themes/10up-theme/tailwind.config.js

```diff
@@ -10,32 +34,60 @@ module.exports = {
 	 */
 	content: [
 		/* Ensure changes to all PHP, JS, and JSON files rebuild your CSS */
-		'./assets/js/**/*.js',
-		'./includes/**/*.{php,js,json}',
-		'./partials/**/*.php',
-		'./templates/**/*.php',
+		'404.php',
+		'archive.php',
+		'attachment.php',
+		'author.php',
+		'category.php',
+		'date.php',
+		'footer.php',
+		'front-page.php',
+		'functions.php',
+		'header.php',
+		'home.php',
+		'index.php',
+		'page.php',
+		'search.php',
+		'searchform.php',
+		'single.php',
+		'singular.php',
+		'tag.php',
+		'taxonomy.php',
+
+		// Directories
+		'assets/**/*.js',
+		'includes/**/*.{js, php}',
+		'partials/**/*.php',
+		'templates/**/*.php',
 	],
 	safelist: [
-		/* Prevent editor-specific styles from being purged */
-		'editor-post-title__block',
-		'editor-post-title__input',
-		'entry-content',
-		'entry-title',
-		'block-editor-block-list__layout',
+		{
+			pattern: /(bg|text)-(primary|secondary|tertiary)/,
+		},
 	],
 	theme: {
 		/* Override the default theme */
-
+		container: {
+			center: true,
+			padding: {
+				default: '1.25rem',
+				md: '2rem',
+				lg: '2.5rem',
+			},
+		},
 		/* Define brand colors using --wp custom properties from the theme.json */
 		colors: {
 			transparent: 'transparent',
 			current: 'currentColor',
-			white: 'var(--wp--preset--color--white)',
```

Comment:
@iansvo will `--wp--preset--color--white` ever be anything other than `#fff` ?

Reply:
@dainemawer While that is the case, the value here is coming from a defined color (e.g. it doesn't just exist by default). You can see this in the Block Editor handbook example here: https://developer.wordpress.org/block-editor/how-to-guides/themes/theme-json/#css-custom-properties-presets-custom.

It is also worth noting that in general, true black/white isn't the color you'd want to use unless it's part of the brand guidelines. More often than not I'm seeing folks using offblack/white (think like `#fefefe` vs `#ffffff`, or `#0a0a0a` instead of `#000000`.